Tone/instrument/Sampler.ts

```diff
@@ -197,6 +226,7 @@ export class Sampler extends Instrument<SamplerOptions> {
 		if (!Array.isArray(notes)) {
 			notes = [notes];
 		}
+        const offset = defaultArg(0, this._loopStart);
```

Comment:
I think you have this reversed, should be 

`const offset = defaultArg(this._loopStart, 0);`